appveyor.yml

```diff
@@ -40,7 +40,8 @@ init:
 install:
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
   - "%CMD_IN_ENV%  pip install setuptools>=18.5 --upgrade"
-  - "%CMD_IN_ENV%  pip install nose coverage ipython[test] --upgrade"
+  - "%CMD_IN_ENV%  pip install nose coverage"
+  - "%CMD_IN_ENV%  pip install .[test]"
 test_script:
-  - "%CMD_IN_ENV%  python -c 'True'"
+  - "%CMD_IN_ENV%  nosetests ."
```

Comment:
This should use our `iptest` command rather than `nosetests`. You may need to cd to a different directory before running it. Have a look at `.travis.yml` for comparison.